lib/locales/en.ts

```diff
@@ -918,6 +951,7 @@ export const locale: Locale = {
     rose_of_lima_virgin: 'Saint Rose of Lima, Virgin',
     rose_of_lima_virgin_copatroness_of_the_philippines: 'Saint Rose of Lima, Virgin, Copatroness of the Philippines',
     rose_philippine_duchesne_virgin: 'Saint Rose Philippine Duchesne, Virgin',
+    rumpharius_of_coutances_bishop: 'Saint Rumpharius, Bishop of Coutances',
```

Comment:
In English celebration names, the bishopric see is omited from after the `Bishop` title.

```suggestion
    rumpharius_of_coutances_bishop: 'Saint Rumpharius, Bishop',
```